frontend/app/tests/unit/specs/composables/history/notes.spec.ts

```diff
@@ -314,27 +302,25 @@ describe('composables::history/notes', () => {
   it('scramble IBAN', () => {
     store.update({ scrambleData: false });
     const iban = 'DE88 5678 9012 1234 345 67';
-    const notes = `Send 8,325.00 EURe via bank transfer to Rotki Solutions GmbH (${iban}) with memo "for salaries and insurance`;
+    const notes = `Send 8,325.00 EURe via bank transfer to Rotki Solutions GmbH (${iban}) with memo "for salaries and insurance"`;
 
     const notesData = formatNotes({ notes, counterparty: 'monerium' });
     let formatted = get(notesData);
     let notesToString = formatted
       .filter(item => item.type === NoteType.WORD)
       .map(item => item.word)
       .join('');
-    let included = notesToString.includes(iban.split(' ').join(''));
 
-    expect(included).toBeTruthy();
+    expect(notesToString).toContain(iban);
 
     store.update({ scrambleData: true });
     formatted = get(notesData);
     notesToString = formatted
       .filter(item => item.type === NoteType.WORD)
       .map(item => item.word)
       .join('');
-    included = notesToString.includes(iban.split(' ').join(''));
 
-    expect(included).toBeFalsy();
+    expect(notesToString).not.toContain(iban);
```

Comment:
I changed the test here so that if something goes wrong we have a more clear message.